hybridse/src/udf/udf.h

```diff
@@ -224,6 +224,10 @@ int32_t weekofyear(int64_t ts);
 int32_t weekofyear(Timestamp *ts);
 int32_t weekofyear(Date *ts);
 
+void last_day(int64_t ts, Date *output, bool *is_null);
+void last_day(Timestamp *ts, Date *output);
+void last_day(Date *ts, Date *output, bool *is_null);
```

Comment:
possible to define as `const Date*` and `const Timestamp*`?

Reply:
Surely that's a better practice. Should I also change other udfs' input pointer param as const? That could be more unified

Reply:
It's ok to just change the `const` for the newly-added functions. 

For other udfs' signature, it would be better you can create a new PR and do the fix, if you are interested. Thanks.